crates/forge/tests/integration/setup_fork.rs

```diff
@@ -160,3 +162,67 @@ fn fork_cairo0_contract() {
 
     assert_passed!(result);
 }
+
+#[test]
+fn get_block_timestamp_in_forked_block() {
```

Comment:
Can we add a test verifying the error returned when using `Pending` block and trying to get block info? Can use `assert_case_output_contains`

One test with `Latest` would be nice too

Reply:
added a test for the Latest block

The problem with the Pending block error is that this error fails the runner